### PR TITLE
Set tentatively good default values in resharding config

### PR DESCRIPTION
**Pull request by Trisfald**
PR to update resharding config to "good" starting values.

### CODE REVIEW

File: core/chain-configs/src/client_config.rs [hunk 1]
```diff
@@ -290,14 +293,14 @@ impl Default for ReshardingConfig {
         // extra load on the node as possible.
         Self {
             batch_size: ByteSize::kb(500),
-            batch_delay: Duration::milliseconds(100),
+            batch_delay: Duration::milliseconds(5),
```
Comment (Trisfald) [inline core/chain-configs/src/client_config.rs:296]: I think this's what @Longarithm has been using in forknet

File: core/chain-configs/src/client_config.rs [hunk 2]
```diff
@@ -264,17 +264,20 @@ pub struct ReshardingConfig {
 
     /// The delay between attempts to start resharding while waiting for the
     /// state snapshot to become available.
+    /// UNUSED in ReshardingV3.
```
Comment (wacban) [inline core/chain-configs/src/client_config.rs:267]: Nice! 

Is there any sort of "deprecated" tag that we could use, that will show a warning if those fields are set?

**Review by wacban — APPROVED** [review]

File: core/chain-configs/src/client_config.rs (hunk 2 above)
Comment (Trisfald) [reply core/chain-configs/src/client_config.rs:267]: I was thinking we could remove these fields entirely? Do you foresee any issue?

Comment (wacban) [reply core/chain-configs/src/client_config.rs:267]: Ah yeah that's even better. I thought you're keeping them for the resharding v2 replayability but as long as we're past that phase let's get rid of it.

**Pull request merged**